<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.2.2 (Python 3.12.5 on win32)" generated="2025-06-09T02:01:55.978575" rpa="false" schemaversion="5">
<suite id="s1" name="WiFiAndConnectivite" source="C:\Users\bahae\Documents\PfeProject\test_cases\WiFiAndConnectivite.robot">
<kw name="Démarrer Driver" type="SETUP">
<kw name="Setup Driver" owner="Integration">
<msg time="2025-06-09T02:01:59.618146" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000020768E6B650&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée')': /session</msg>
<msg time="2025-06-09T02:02:01.657090" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000020768E6B740&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée')': /session</msg>
<msg time="2025-06-09T02:02:03.687188" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000020768E6B9E0&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée')': /session</msg>
<msg time="2025-06-09T02:02:05.710480" level="FAIL">MaxRetryError: HTTPConnectionPool(host='host.docker.internal', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000020768E6BC80&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))</msg>
<var>${driver}</var>
<arg>${Device}</arg>
<doc>Initialise et retourne le driver Appium.</doc>
<status status="FAIL" start="2025-06-09T02:01:57.566174" elapsed="8.184894">MaxRetryError: HTTPConnectionPool(host='host.docker.internal', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000020768E6BC80&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))</status>
</kw>
<kw name="Set Suite Variable" owner="BuiltIn">
<arg>${driver}</arg>
<doc>Makes a variable available everywhere within the scope of the current suite.</doc>
<status status="NOT RUN" start="2025-06-09T02:02:05.752085" elapsed="0.000000"/>
</kw>
<kw name="Initialize Language Settings">
<doc>Sets up language-specific variables based on system language</doc>
<status status="NOT RUN" start="2025-06-09T02:02:05.752085" elapsed="0.000000"/>
</kw>
<doc>Initializes the test driver</doc>
<status status="FAIL" start="2025-06-09T02:01:57.566174" elapsed="8.185911">MaxRetryError: HTTPConnectionPool(host='host.docker.internal', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000020768E6BC80&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))</status>
</kw>
<test id="s1-t1" name="Test Ouvrir Wifi" line="143">
<doc>Teste l'activation du WiFi
- Ouvre les paramètres
- Active le WiFi
- Vérifie l'activation via UI et ADB</doc>
<tag>smoke</tag>
<tag>wifi</tag>
<status status="FAIL" start="2025-06-09T02:02:05.752085" elapsed="0.000984">Parent suite setup failed:
MaxRetryError: HTTPConnectionPool(host='host.docker.internal', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000020768E6BC80&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))</status>
</test>
<kw name="Fermer Driver" type="TEARDOWN">
<try>
<branch type="TRY">
<kw name="Run Keyword And Return Status" owner="BuiltIn">
<kw name="Get Source" owner="AppiumLibrary">
<kw name="Capture Page Screenshot" owner="AppiumLibrary">
<msg time="2025-06-09T02:02:05.755069" level="FAIL">No application is open</msg>
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<status status="FAIL" start="2025-06-09T02:02:05.754069" elapsed="0.014831">No application is open</status>
</kw>
<msg time="2025-06-09T02:02:05.769894" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg time="2025-06-09T02:02:05.769894" level="FAIL">No application is open</msg>
<doc>Returns the entire source of the current page.</doc>
<status status="FAIL" start="2025-06-09T02:02:05.754069" elapsed="0.015825">No application is open</status>
</kw>
<msg time="2025-06-09T02:02:05.770893" level="INFO">${session_active} = False</msg>
<var>${session_active}</var>
<arg>Get Source</arg>
<doc>Runs the given keyword with given arguments and returns the status as a Boolean value.</doc>
<status status="PASS" start="2025-06-09T02:02:05.754069" elapsed="0.016824"/>
</kw>
<if>
<branch type="IF" condition="${session_active}">
<kw name="Close Driver" owner="Integration">
<arg>${driver}</arg>
<doc>Ferme proprement le driver Appium.</doc>
<status status="NOT RUN" start="2025-06-09T02:02:05.770893" elapsed="0.000000"/>
</kw>
<kw name="Log" owner="BuiltIn">
<arg>Driver fermé avec succès</arg>
<doc>Logs the given message with the given level.</doc>
<status status="NOT RUN" start="2025-06-09T02:02:05.770893" elapsed="0.000000"/>
</kw>
<status status="NOT RUN" start="2025-06-09T02:02:05.770893" elapsed="0.000000"/>
</branch>
<branch type="ELSE">
<kw name="Log" owner="BuiltIn">
<msg time="2025-06-09T02:02:05.770893" level="INFO">Le driver n'était pas actif</msg>
<arg>Le driver n'était pas actif</arg>
<doc>Logs the given message with the given level.</doc>
<status status="PASS" start="2025-06-09T02:02:05.770893" elapsed="0.000000"/>
</kw>
<status status="PASS" start="2025-06-09T02:02:05.770893" elapsed="0.000000"/>
</branch>
<status status="PASS" start="2025-06-09T02:02:05.770893" elapsed="0.000000"/>
</if>
<status status="PASS" start="2025-06-09T02:02:05.754069" elapsed="0.016824"/>
</branch>
<branch type="EXCEPT" assign="${error}">
<kw name="Log" owner="BuiltIn">
<arg>Erreur lors de la fermeture du driver: ${error}</arg>
<doc>Logs the given message with the given level.</doc>
<status status="NOT RUN" start="2025-06-09T02:02:05.770893" elapsed="0.000000"/>
</kw>
<status status="NOT RUN" start="2025-06-09T02:02:05.770893" elapsed="0.000000"/>
</branch>
<status status="PASS" start="2025-06-09T02:02:05.754069" elapsed="0.016824"/>
</try>
<doc>Cleans up and closes the test driver</doc>
<status status="PASS" start="2025-06-09T02:02:05.753069" elapsed="0.017824"/>
</kw>
<doc>Tests de connectivité WiFi pour appareil Android
- Activation du WiFi
- Désactivation du WiFi
- Vérification de l'état via UI et ADB
- Prend en charge les interfaces en français et en anglais</doc>
<status status="FAIL" start="2025-06-09T02:01:55.980567" elapsed="9.790326">Suite setup failed:
MaxRetryError: HTTPConnectionPool(host='host.docker.internal', port=4723): Max retries exceeded with url: /session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000020768E6BC80&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))</status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
<stat pass="0" fail="1" skip="0">smoke</stat>
<stat pass="0" fail="1" skip="0">wifi</stat>
</tag>
<suite>
<stat name="WiFiAndConnectivite" id="s1" pass="0" fail="1" skip="0">WiFiAndConnectivite</stat>
</suite>
</statistics>
<errors>
<msg time="2025-06-09T02:01:59.618146" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000020768E6B650&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée')': /session</msg>
<msg time="2025-06-09T02:02:01.657090" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000020768E6B740&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée')': /session</msg>
<msg time="2025-06-09T02:02:03.687188" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000020768E6B9E0&gt;: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée')': /session</msg>
<msg time="2025-06-09T02:02:05.769894" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
</errors>
</robot>
